src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +124,139 @@ end
 end bounded
 
 end number_field.embeddings
+
+section place
+
+variables {A : Type*} [normed_division_ring A] {K : Type*} [field K] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def number_field.place : absolute_value K ℝ :=
+{ to_fun := norm ∘ φ,
+  map_mul' := by simp only [function.comp_app, map_mul, norm_mul, eq_self_iff_true, forall_const],
+  nonneg' := by simp only [norm_nonneg, forall_const],
+  eq_zero' := by simp only [norm_eq_zero, map_eq_zero, forall_const, iff_self],
+  add_le' := by simp only [function.comp_app, map_add, norm_add_le, forall_const], }
+
+end place
+
+namespace number_field.complex_embeddings
+
+open complex number_field
+
+open_locale complex_conjugate
+
+variables {K : Type*} [field K]
+
+/-- The conjugate of a complex embedding as a complex embedding. -/
+def conjugate (φ : K →+* ℂ) : K →+* ℂ := ring_hom.comp conj_ae.to_ring_equiv.to_ring_hom φ
```

Comment:
Yes, that is done. But the following result seems to be missing (it would give a direct proof of `is_real_conjugate_iff`)
```lean
lemma is_self_adjoint_star_iff {R : Type*} [has_involutive_star R] {x : R} :
  is_self_adjoint (has_star.star x) ↔ is_self_adjoint x :=
by simpa only [is_self_adjoint, star_star] using eq_comm
```
Should I PR it?

Reply:
> Yes, that is done. But the following result seems to be missing (it would give a direct proof of `is_real_conjugate_iff`)
> 
> ```lean
> lemma is_self_adjoint_star_iff {R : Type*} [has_involutive_star R] {x : R} :
>   is_self_adjoint (has_star.star x) ↔ is_self_adjoint x :=
> by simpa only [is_self_adjoint, star_star] using eq_comm
> ```
> 
> Should I PR it?

If it's simple enough you can add to this PR, if it's more than a couple of lines feel free to open a new one.